android/source/src/main/kotlin/com/gu/source/utils/PreviewAnnotations.kt

```diff
@@ -1,5 +1,3 @@
-@file:Suppress("ktlint:compose:preview-annotation-naming")
```

Comment:
Naming convention is to begin the name with `Preview`. We were suppressing that lint error so far. Now we've got the naming correct so can remove the suppression.